src/main/java/com/google/sps/utility/TasksUtility.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.utility;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.tasks.Tasks;
+import com.google.api.services.tasks.model.Task;
+import com.google.api.services.tasks.model.TaskList;
+import java.io.IOException;
+import java.util.List;
+
+public final class TasksUtility {
```

Comment:
Overall, I am concerned this pattern of creating utility methods will become unmanageable. I think `getTasksService` is a good utility function, but `listTaskLists` and `listTasks` can be problematic.
 - The number of utility can become large, if you need to support different options for listing things.
 - These methods are hard to mock in tests, because you have to supply a mock of this complex Tasks service.

The point about testing is the real issue in my opinion.

Reply:
This will be changed in another PR #39

Reply:
This will be handled in the next PR #39